addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -415,21 +429,21 @@
         <field name="model">hr.job</field>
         <field name="priority">200</field>
         <field name="arch" type="xml">
-            <form string="Create a Job Position">
+            <form string="Create a Job Position" class="o_hr_job_simple_form" >
                 <sheet>
                     <group>
-                        <field name="name" class="oe_inline" placeholder="e.g. Sales Manager"/>
+                        <field name="name" class="o_job_name oe_inline" placeholder="e.g. Sales Manager"/>
                         <label for="alias_name" string="Application email" attrs="{'invisible': [('alias_domain', '=', False)]}" help="Define a specific contact address for this job position. If you keep it empty, the default email address will be used which is in human resources settings"/>
                         <div name="alias_def" attrs="{'invisible': [('alias_domain', '=', False)]}">
                             <field name="alias_id" class="oe_read_only" string="Email Alias" required="0"/>
                             <div class="oe_edit_only" name="edit_alias">
-                                <field name="alias_name" class="oe_inline"/>@<field name="alias_domain" class="oe_inline" readonly="1"/>
+                                <field name="alias_name" class="oe_inline o_job_alias" placeholder="e.g. sales-manager"/>@<field name="alias_domain" class="oe_inline" readonly="1"/>
```

Comment:
This one too. They look like a remaining dead development 

Could you please take a look at the whole PR and remove the dead classes ?